source/vibe/data/json.d

```diff
@@ -181,6 +199,8 @@ struct Json {
 	/// ditto
 	long opAssign(long v) { m_type = Type.int_; m_int = v; return v; }
 	/// ditto
+	BigInt opAssign(BigInt v) { m_type = Type.bigint; initBigInt(); m_bigint = v; return v; }
```

Comment:
>IDK how BigInt are implemented, but I can see the cycle

Default value may be only static value, because BigInt is a struct, and BigInt.init should be known at compile time.
Thus everytime, `a`, `b` and `c` in the following example has a identical (bitwise) value. Thus I simply write BigInt.init to the m_data.
```
    BigInt a;
    BigInt b;
    BigInt c;
```